FirstClasses/average/src/average.erl

```diff
@@ -3,4 +3,49 @@
 -export([average/1]).
 
 average(_List) ->
-    ok.
+    case _List of 
+        [] ->
+            0;
+        _ ->
+            sum(_List) / list_lenght(_List)
+    end.
+
+sum(List) ->
+    %sum_buildin(List).
```

Comment:
I suggest if you are not using those, you might just delete them. 
Kinda confusing to follow. 
What I mean is that since you commented those `*_buildin` calls probably would be better just to remove those lines + the implementations